internify/src/store/reducers/userReducer.js

```diff
@@ -0,0 +1,37 @@
+import {
+  ADD_USER,
+  REMOVE_USER,
+  UPDATE_USER,
+  GET_USERS,
+} from "../actions/types/userTypes";
+
+const initialState = {
+  userList: [],
+};
+
+export default function userReducer(state = initialState, action) {
+  switch (action.type) {
+    case ADD_USER:
+      const newUser1 = action.payload.user;
+      return [...state, newUser1];
```

Comment:
why did we decide to use array list as opposed to returning objects? let's discuss what type we want to pick and stick to it throughout the code

Reply:
Lets definitely talk about this - really what I did was follow whatever framework was in the job reducer.

Reply:
^ resolved - will keep as list for now